docs/src/appendix/snfoundry-toml.md

```diff
@@ -0,0 +1,103 @@
+# The Manifest Format
+
```

Comment:
I think we should also say what's the another purpose of this file: You can configure some arguments in it to not have to provide them in CLI along with the commands.